app/views/admin/accounts/show.html.haml

```diff
@@ -1,6 +1,10 @@
 - content_for :page_title do
   = @account.pretty_acct
 
+- content_for :heading_actions do
+  - unless @account.local?
+    = link_to t('admin.accounts.view_instance'), admin_instance_path(@account.domain), method: :post, class: 'button'
```

Comment:
Is the `method: :post` here intentional?

Reply:
It is not, good catch.

Reply:
```suggestion
    = link_to t('admin.accounts.view_instance'), admin_instance_path(@account.domain), class: 'button'
```